api/nodes/views.py

```diff
@@ -1660,7 +1662,14 @@ def get_queryset(self):
 
     # overrides ListCreateAPIView
     def perform_create(self, serializer):
-        serializer.save(node=self.get_node())
+        user = get_user_auth(self.request).user
+        node = self.get_node()
+        try:
+            fork = serializer.save(node=node)
+            mails.send_mail(user.email, mails.FORK_COMPLETED, title=fork.title, guid=fork._id, mimetype='html')
```

Comment:
Also, it might make sense to use the try-except-else so we don't accidentally send a failed fork email if the fork_completed email fails.